### PR TITLE
ICU-21480 BRS69 ICU4J API signature file

### PR DESCRIPTION
**Pull request by yumaoka**
- Supplied missing API comment to com.ibm.icu.nubmer.Precision#trailingZeroDisplay(TrailingZeroDisplay)
- Generated API signature file for ICU4J 69.1 and added to the repository
- Delete ICU4J 59 API signature file
- Deleted unused icu4j/eclipse-build/build.properties. The folder eclipse-build was deleted, but someone worked on ICU version seemed to add it back again. Delete it again.

##### Checklist

- [x] Required: Issue filed: https://unicode-org.atlassian.net/browse/ICU-21480
- [x] Required: The PR title must be prefixed with a JIRA Issue number. 
- [x] Required: The PR description must include the link to the Jira Issue, for example by completing the URL in the first checklist item
- [x] Required: Each commit message must be prefixed with a JIRA Issue number. 
- [x] Issue accepted (done by Technical Committee after discussion)
- [ ] Tests included, if applicable
- [ ] API docs and/or User Guide docs changed or added, if applicable

### CODE REVIEW

**Review by sffc — APPROVED** [review]
Thanks for doing this, and sorry for leaving out the docstring!  Did you copy it from the C++ version?

File: icu4j/main/classes/core/src/com/ibm/icu/number/Precision.java [hunk 1]
```diff
@@ -339,6 +339,13 @@ public static CurrencyPrecision currency(CurrencyUsage currencyUsage) {
         }
     }
 
+    /**
+     * Configure how trailing zeros are displayed on numbers. For example, to hide trailing zeros
+     * when the number is an integer, use HIDE_IF_WHOLE.
+     *
+     * @param trailingZeroDisplay Option to configure the display of trailing zeros.
+     * @draft ICU 69
```
Comment (sffc) [inline icu4j/main/classes/core/src/com/ibm/icu/number/Precision.java:347]: `@provisional`?

Comment (yumaoka) [reply icu4j/main/classes/core/src/com/ibm/icu/number/Precision.java:347]: We no longer use @provisional tag. See https://unicode-org.atlassian.net/browse/ICU-21369

**Comment by yumaoka** [comment]
> Thanks for doing this, and sorry for leaving out the docstring! Did you copy it from the C++ version?

Copied C++ version, but changed the enum name to match ICU4J definition.

**Review by markusicu — APPROVED** [review]
lgtm

**Pull request merged**